recipes/gts/config.in.win32.patch

```diff
@@ -0,0 +1,13 @@
+--- src/config.h.win32
++++ src/config.h.win32
+@@ -7,8 +7,8 @@
+ #define STDC_HEADERS 1
+ 
+ #define GTS_MAJOR_VERSION 0
+-#define GTS_MINOR_VERSION 6
+-#define GTS_MICRO_VERSION 0
++#define GTS_MINOR_VERSION 7
++#define GTS_MICRO_VERSION 6

```

Comment:
Is the version number really wrong in the source code?

Reply:
Yeah, it does have a `config.in` as a template but it is for non-Windows, but the Windows one `config.in.win32` appears to be a hand-crafted one and out-of-dated on the version...

Reply:
Weird.... you should file a bug

Reply:
Filed one...https://sourceforge.net/p/gts/bugs/73/